app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> {order(created_at: :desc)}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
vadidates kiểm tra độ hợp lệ các thuộc tính trong modle ví dụ như name, email, ... sử dụng các thuộc tính có sẵn của rails như: presence, uniqueness, length, ...
vadidate kiểm tra độ hợp lệ của phương thức mà ta đã xây dưng. ví dụ.

validate :picture_size
def picture_size
    if picture.size > 5.megabytes
      errors.add :picture, "should be less than 5MB"
    end